build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -21,31 +23,35 @@ prepare_cluster_for_gpu_operator() {
 }
 
 collect_must_gather() {
-    set +x
-    echo "Running gpu-operator_gather ..."
-    /usr/bin/gpu-operator_gather &> /dev/null
+    run_in_sub_shell() {
```

Comment:
Function name is a bit misleading, it sounds like this function will run something in a sub shall, rather than be ran in a sub shell itself

Reply:
Maybe rename it to `collect` or something

Reply:
don't know, it's a local function witht the subshell invocation a few lines after, I don't find it hard to understand ...
```
run_in_sub_shell() {
...
}
    # run the function above in a subshell to avoid polluting the local `env`.
    typeset -fx run_in_sub_shell
    bash -c run_in_sub_shell
```
that's similar to 
```
cb() {
 ... do callback ...
}
iterate_over_something(cb)
```

Reply:
I personally was mislead / confused by it, so I thought I should explain why. Doesn't really matter